src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,13 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        if (email == "bob@i.ua") {
```

Comment:
you should use vise verse algorithm
use loop for iterating users
` for (int i = 0; i < users.length; i++) {`

and inside this loop check which `user` has `email`  that equals `email` from method input parameter